packages/core/src/utils/TypeUtils.ts

```diff
@@ -54,7 +54,7 @@ export interface DataWallet {
 
 export interface ApiGetWalletsRequest {
   page: number;
-  entries: number;
+  entries?: number;
```

Comment:
`entries` is required param in query https://specs.walletconnect.com/2.0/specs/meta-clients/web3modal/api#get-getwallets